lib/sqlalchemy/testing/reflection_fixture.py

```diff
@@ -0,0 +1,201 @@
+from . import config
+from .assertions import eq_
+from .assertions import is_
+from .assertions import is_not_
+from .assertions import is_true
+from .fixtures import TablesTest
+from .. import inspect
+from .. import Integer
+from ..schema import Column
+from ..schema import Computed
+from ..schema import MetaData
+from ..schema import Table
+
+
+class ComputedReflectionFixtureTest(TablesTest):
+    run_inserts = run_deletes = None
+
+    __backend__ = True
+    __requires__ = ("computed_columns", "table_reflection")
+
+    return_persisted = None
+    default_persisted = None
+    support_stored = None
+    support_virtual = None
+
+    def to_sqltext(self, column, op, value):
```

Comment:
I think for now we can likely just use a regex that filters out brackets and parenthesis, and just set this up once for all the current dialects, right?  that should be good enough.